src/components/molecules/TextView/TextViewRecordLargeNumbers.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Box from '@material-ui/core/Box';
+import { roadIconColors, silverSmokeColor } from 'style';
+import SeverityImage from './SeverityImage';
+import { makeStyles } from '@material-ui/core/styles';
+import { Typography } from 'components/atoms';
+
+interface IProps {
+  numOfAccidents: number;
+  severityDesc: string;
+  imgSrc: string;
+  isLast?: boolean;
+}
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: 'flex',
+    justifyContent: 'space-around',
+    width: '80%',
+    height: '20%',
+  },
+  text: {
+    top: theme.spacing(2),
+    display: 'flex',
+    alignItems: 'center',
+    textAlign: 'center',
+    justifyContent: 'center',
+    width: '30%',
+  },
+  acNum: {
+    position: 'relative',
+    display: 'flex',
+    alignItems: 'center',
+    textAlign: 'center',
+    justifyContent: 'center',
+    color: roadIconColors.red,
+    fontWeight: 'bold',
+    fontSize: '350%',
+    width: '30%',
+  },
+}));
+
+const TextViewRecordLargeNumbers: React.FC<IProps> = ({ isLast, numOfAccidents, severityDesc, imgSrc }) => {
+  const classes = useStyles();
+  return (
+    <Box className={classes.root} py={1} borderBottom={isLast ? '' : `5px solid ${silverSmokeColor}`}>
```

Comment:
try to use `use styles()` with parameters instead of inline styles for the `borderBottom` and the `py`
example in https://www.codegrepper.com/code-examples/whatever/how+to+pass+variables+to+makestyles.
also, you can use `:last-child`  CSS property to set the border instead of asking `isLast`
see here:https://stackoverflow.com/questions/50602582/use-css-last--change-border